src/Rss/FeedRss.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+namespace Bonnier\WP\ContentHub\Editor\Rss;
+
+use App\Enum\AcfName;
+use App\Helpers\SortByHelper;
+use App\Models\Views\CompositeTeaser;
+use Bhaktaraz\RSSGenerator\Channel;
+use Bhaktaraz\RSSGenerator\Feed;
+use Bonnier\WP\ContentHub\Editor\Models\ACF\Composite\CompositeFieldGroup;
+use Bonnier\WP\ContentHub\Editor\Models\WpAttachment;
+use Bonnier\WP\ContentHub\Editor\Models\WpComposite;
+use Bonnier\WP\ContentHub\Editor\Singleton;
+use Illuminate\Support\Collection;
+use Parsedown;
+use WP_Post;
+use WP_Term;
+
+class FeedRss
+{
+    /**
+     * @var Feed $feed
+     */
+    private $feed;
+
+    /**
+     * @var Channel $channel
+     */
+    private $channel;
+
+    use Singleton;
+
+    public static function register()
+    {
+        remove_action('do_feed_rss2', 'do_feed_rss2', 10);
+        add_action('do_feed_rss2', [self::getInstance(), 'generateOutput']);
+    }
+
+    public function __construct()
+    {
+        $this->feed = new MsnFeed();
+        $this->channel = new Channel();
+    }
+
+    public function makeChannel()
+    {
+        $frontPageId = get_option('page_on_front');
+        $homeLink = get_permalink($frontPageId);
+        $this->channel
+            ->title(wp_get_document_title())
+            ->description(get_post_meta($frontPageId, '_yoast_wpseo_metadesc', true))
+            ->atomLinkSelf($homeLink)
+            ->url($homeLink)
+            ->language(get_bloginfo_rss('language'))
+            ->appendTo($this->feed);
+    }
+
+    /**
+     * Generate final MSN Rss feed output
+     */
+    public function generateOutput()
+    {
+        $this->makeChannel();
+        $this->getPosts()->each(function (WP_Post $composite) {
+            $this->makeItem($composite);
+        });
+
+        header('Content-Type: Application/xml');
+        echo $this->feed;
```

Comment:
Burde du ikke kalde `wp_die()` efter `echo` for at sikre, at der ikke bliver outputtet andet i rss feeded?